src/components/nodes/components/NodeUrl.vue

```diff
@@ -5,7 +5,9 @@
 
   <template v-else>
     <span :class="classes.nodeName">{{ nodeHost.name }}</span>
-    <span :class="classes.domain">.{{ nodeHost.domain }}</span>
+    <span v-if="nodeHost.domain && nodeHost.domain.length > 0" :class="classes.domain"
```

Comment:
Is `nodeHost.domain.length > 0` required here? I think checking if `nodeHost.domain` is truthy should be enough.